lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -952,6 +953,17 @@ def _create_deployment_parameters(
             parameters=parameters,
         )
 
+        # dump arm_template and arm_parameters to file
+        dump_path = constants.RUN_LOCAL_PATH / "arm_dump"
+        try:
+            os.mkdir(dump_path)
+        except OSError:
+            pass
```

Comment:
it's not a good practice to eat an exception without more strict limitation or logging. If it's used to handle path existing situation, there is an argument `exist_ok` on `mkdir`.